deployer/deployer.py

```diff
@@ -236,11 +237,14 @@ def deploy_version(self, version):
             }
         }
 
-        r = self.request("deployment/start", json.dumps(data))
+        for i in range(1, 10):
```

Comment:
@Arcontus We need to talk more about this, I don't like too much this way, you don't need to put a for in this point because if the response it's ok you have a 200 code, if not you have another code. so that you need to know the response code, maybe if the response is a 4xx you can retry but it's a strange case.

I understand what you are trying to do but maybe it's not the best way (before do this is better we talk before), I know that even if the asgard answer is 200 the deploy can fails but it's better to wait the maximum timeout (you have the param self.start_up_timeout_minutes) and we can folow the deploy with the URL:

http://asgard.global-pre.spain.schibsted.io:8080/task/show/1233.json
http://asgard.global-pre.spain.schibsted.io:8080/task/show/1234.json

that have the real status... the task number is in the response of the deploy action, you can see the output in the jenkins output:

http://jenkins.spain.schibsted.io:8080/view/devops-common--ms-testing/view/pipeline/job/devops-common--ms-testing_DEPLOY_PRE/44/console

the line with the json:
```
{
  "deploymentId": "1233"
}
```
corresponds with the line 241 you removed of this file.